modules/connectivity/locals.tf

```diff
@@ -1288,6 +1288,8 @@ locals {
     azure_sql_database_sqlserver         = ["privatelink.database.windows.net"]
     azure_synapse_analytics_sqlserver    = ["privatelink.database.windows.net"]
     azure_synapse_analytics_sql          = ["privatelink.sql.azuresynapse.net"]
+    azure_synapse                        = ["privatelink.azuresynapse.net"]
```

Comment:
Would this be better referred to as `azure_synapse_studio`?

@mboswell and @marvinbuss please can you input on this one, but also a broader view on the current state of our data product coverage? i.e. are the `azure_synapse_analytics_sqlserver` and `azure_synapse_analytics_sql` entries still relevant as the former no longer seems to be mentioned in the docs? 😄

Reply:
Hi @krowlandson,
Sorry, I just saw that you tagged me in this comment. `privatelink.database.windows.net` is not relevant for Synapse. This is only relevant for Azure SQL Servers.

Synapse only requires:
1. For SQL endpoints: `privatelink.sql.azuresynapse.net`
2. For dev endpoints: `privatelink.dev.azuresynapse.net`
3. For Data Explorer endpoints: `privatelink.kusto.azuresynapse.net`
4. For Synapse Studio: `privatelink.azuresynapse.net`

Feel free to rename one of the variables to `azure_synapse_studio` accordingly.

Reply:
Hi @krowlandson,
Sorry, I just saw that you tagged me in this comment. `privatelink.database.windows.net` is not relevant for Synapse. This is only relevant for Azure SQL Servers.

Synapse only requires:
1. For SQL endpoints: `privatelink.sql.azuresynapse.net`
2. For dev endpoints: `privatelink.dev.azuresynapse.net`
3. For Data Explorer endpoints: `privatelink.kusto.azuresynapse.net`
4. For Synapse Studio: `privatelink.azuresynapse.net`

Feel free to rename one of the variables to `azure_synapse_studio` accordingly.

Reply:
Cool, thank you for the confirmation @marvinbuss 👍🏻 

I thought it was odd that the docs were updated, as the module reflects what the docs said at the time of authoring this capability, so we definitely need to fix this then 😄